packages/fxa-auth-server/lib/oauthdb/utils.js

```diff
@@ -58,6 +58,8 @@ module.exports = {
       return error.staleAuthAt(err.authAt);
     case 120:
       return error.insufficientACRValues(err.foundValue);
+    case 201:
+      return error.serviceUnavailable(err.retryAfter);
```

Comment:
Strange that `serviceUnavailable` returns [SERVER_BUSY](https://github.com/mozilla/fxa/blob/dc06472fd5278f179065a75779fdfe4baa4efa39/packages/fxa-auth-server/lib/error.js#L497), that's what we show [on the frontend too](https://github.com/mozilla/fxa/blob/dc06472fd5278f179065a75779fdfe4baa4efa39/packages/fxa-content-server/app/scripts/lib/auth-errors.js#L255). I wonder how we can do better.